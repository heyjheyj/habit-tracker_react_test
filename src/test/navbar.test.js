import Navbar from "../Components/navbar";
import React from "react";
import renderer from "react-test-renderer";

describe("Navbar", () => {
  it("renders", () => {
    const Component = renderer.create(<Navbar totalCount={4} />);
    expect(Component.toJSON()).toMatchSnapshot();
  });
});
