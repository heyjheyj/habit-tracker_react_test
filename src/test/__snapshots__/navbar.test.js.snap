// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar renders 1`] = `
<section
  className="Navbar"
>
  <i
    className="fab fa-pagelines"
  />
  <span
    className="title"
  >
    Habit Tracker
  </span>
  <span
    className="habit-count"
  >
    4
  </span>
</section>
`;
